paraatm/tests/aviationr_example.py

```diff
@@ -0,0 +1,52 @@
+import os
+import numpy as np
+import pandas as pd
+from paraatm.simulation_method.aviationr import AviationRisk
+import matplotlib.pyplot as plt
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+data_dir = os.path.join(cur_dir, '..', 'sample_data/')
+
+cfg = {'fp_file': data_dir + 'aviationR/data/TRX_DEMO_SFO_PHX_GateToGate.trx',  # flight plan file
+       'mfl_file': data_dir + 'aviationR/data/TRX_DEMO_SFO_PHX_mfl.trx',  # mfl file
+       'data_file': data_dir + 'aviationR/data/', # required data for accident simulation and risk estimation
+       'model_file': data_dir + 'aviationR/model/', # Pre-trained model
+       'sim_time': 12000}  # total simulation time
+
+# call
+sim = AviationRisk(cfg)
+try:
+    import torch
+except ImportError:
+    print('aviation risk module requires installation of torch')
+    raise
```

Comment:
I suggest removing this.  My original intent was for this message to be in your module, not your example.  Also, don't worry about adding this check to your module.  I am thinking that the import error will be clear enough anyway.

Reply:
Got it.